docs/en/connector-v2/source/MyHours.md

```diff
@@ -0,0 +1,177 @@
+# My Hours
+
+> My Hours source connector
+
+## Description
+
+Used to read data from My Hours.
+
+## Key features
+
+- [x] [batch](../../concept/connector-v2-features.md)
+- [x] [stream](../../concept/connector-v2-features.md)
```

Comment:
When will `stream` be used? And for a SaaS service, how can we get the Incremental data when use `stream` job?

Reply:
done.